GymEventManager/Controllers/Event Details View/EventDetailsViewController.swift

```diff
@@ -8,37 +8,53 @@
 import UIKit
 
 class EventDetailsViewController: UIViewController {
-
-    var event: Event?
+    
+    // MARK: - IBOutlet Variables
     @IBOutlet weak var eventNameLabel: UIButton!
     @IBOutlet weak var tableView: UITableView!
+    
+    // MARK: - Properties
+    var event: Event?
+
+    // MARK: - LifeCycle Functions
     override func viewDidLoad() {
-        super.viewDidLoad()
+        setupView()
+    }
+   
+    private func setupView() {
+        setupTableView()
+        setupEventNameLabel()
+    }
+    
+    private func setupEventNameLabel() {
+        if let currentEvent = event {
+            eventNameLabel.titleLabel?.text?.append(currentEvent.name)
+        }
+    }
+    
+    private func setupTableView() {
         tableView.dataSource = self
         tableView.delegate = self
-        self.tableView.rowHeight = UITableView.automaticDimension
+        tableView.rowHeight = UITableView.automaticDimension
         tableView.register(UINib(nibName: "TableViewCell", bundle: .main), forCellReuseIdentifier: "TableCell")
         tableView.register(UINib(nibName: "HeaderCell", bundle: .main), forCellReuseIdentifier: "HeaderCell")
-
-        if let currentEvent = event {
-            eventNameLabel.titleLabel?.text?.append(currentEvent.name)
-        }
     }
-    @IBAction func backPressed(_ sender: UIButton) {
+    
+    // MARK: - Actions
+    @IBAction private func backPressed(_ sender: UIButton) {
        navigationController?.popViewController(animated: true)
     }
 
 }
 // MARK: - TableView Methods
-
 extension EventDetailsViewController: UITableViewDelegate, UITableViewDataSource {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         let eventsSize = (event?.teams?.count ?? 0)
         return eventsSize+1
     }
+    
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         if indexPath.row == 0 {
-            // nib cell
             if let tCell = tableView.dequeueReusableCell(withIdentifier: "HeaderCell", for: indexPath) as? HeaderCell {
```

Comment:
you can separate cells creation in two different functions for readability. 
also please don't use appreviations like t i don't know what it stands for


`
 func getHeaderCell() -> UITableViewCell {
        let cell = tableView.dequeueReusableCell(withIdentifier: "HeaderCell") as! HeaderCell
        cell.configure(event: event)
        return cell
    }
`